packages/flutter_tools/test/integration.shard/overall_experience_test.dart

```diff
@@ -108,16 +117,22 @@ void main() {
         'called main',
         'called paint',
       ]);
-      expect(result.stdout.where((String line) => !line.startsWith('called ')), <Object>[
-        // logs start after we receive the response to sending SIGUSR1
-        'Performing hot reload...'.padRight(progressMessageWidth),
-        startsWith('Reloaded 0 libraries in '),
-        'Performing hot restart...'.padRight(progressMessageWidth),
-        startsWith('Restarted application in '),
-        '', // this newline is the one for after we hit "q"
-        'Application finished.',
-        'ready',
-      ]);
+      expect(
+        result.stdout.where((String line) => !line.startsWith('called ')),
+        <Object>[
+          // logs start after we receive the response to sending SIGUSR1
+          'Performing hot reload...'.padRight(progressMessageWidth),
+          startsWith('Reloaded 0 libraries in '),
+          'Performing hot restart...'.padRight(progressMessageWidth),
+          startsWith('Restarted application in '),
+          '', // this newline is the one for after we hit "q"
+          'Application finished.',
+          'ready',
+        ],
+        reason: 'stdout from command ${command.join(' ')} was unexpected, '
+          'full Stdout:\n\n${result.stdout.join('\n')}\n\n'
+          'Stderr:\n\n${result.stderr.join('\n')}',
+      );
```

Comment:
I've also been seeing the test below this one failing with the same error, might want to add it there too?

`flutter run can hot reload and hot restart, handle "p" key`

Reply:
ack, and done